actionpack/lib/abstract_controller/callbacks.rb

```diff
@@ -1,4 +1,24 @@
 module AbstractController
+  # = Abastract Controller Callbacks
+  #
+  # Abastract Controller provides hooks during the life cycle of a cotroller action. Callbacks allow you
+  # to trigger logic during the life cycle of a controller action. Available callbacks are:
```

Comment:
Maybe you could say "this cycle" to avoid the repetition with the previous sentence, what do you think ?